apisix/plugins/prometheus/exporter.lua

```diff
@@ -95,23 +95,23 @@ function _M.init()
             "Etcd modify index for APISIX keys",
             {"key"})
 
-    -- per service
+    -- per service and consumer
     metrics.status = prometheus:counter("http_status",
             "HTTP status codes per service in APISIX",
-            {"code", "route", "service", "node"})
+            {"code", "route", "service", "consumer", "node"})
```

Comment:
Perhaps more code comments are better for new comer just like me to understand codes :)

Reply:
@liuxiran more detailed comment is added, thanks for your suggestions.